dvc/command/base.py

```diff
@@ -57,7 +57,7 @@ def default_targets(self):
 
     # Abstract methods that have to be implemented by any inheritance class
     def run(self):
-        pass
+        raise NotImplementedError("Abstract")
```

Comment:
```suggestion
        raise NotImplementedError
```
should be enough and conforms to the rest of the codebase :slightly_smiling_face: E.g. `dvc/remote/base.py`